src/environments/environment.ts

```diff
@@ -238,7 +238,8 @@ export const environment: ValtimoConfig = {
     showUserNameInTopBar: true,
     showPlantATreeButton: true,
     experimentalDmnEditing: true,
-    caseListColumn: true
+    caseListColumn: true,
+    largeLogoMargin: true,
```

Comment:
A featureToggle seems like an overkill. Implementations can do the same but with more options in styles.scss